generate/generate.go

```diff
@@ -1,3 +1,4 @@
+// Package generate demonstrates how to use moq with the go:generate directive.
```

Comment:
Note that I wrote the same description for this package and the `example` package. It's actually not clear to me what the difference is between these two packages.